.circleci/config.yml

```diff
@@ -207,67 +207,71 @@ commands:
 jobs:
     test-chromium:
         executor: node
+        parallelism: 5
 
         steps:
             - downstream
             - run:
-                  name: Run unit tests
-                  command: yarn test:ci --config web-test-runner.config.ci-chromium.js --group no-memory-ci --coverage
+                  name: Run chromium tests
```

Comment:
all unit tests now run in parallel under 4 min so we will get tests failing faster and not be slowed down by coverage